### PR TITLE
minor material-ui related tweaks

### PR DESCRIPTION
**Pull request by eps1lon**
`master` doesn't list you yet. Looks probably weird linking to it and not being listed for an outsider

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+By the way, I am a fan of open-source. Now I am a collaborator of [Material-UI](https://next.material-ui.com/discover-more/team/) - the most popular UI kit for React in the world. Creator + maintainer of [material-ui-pickers](https://material-ui-pickers.dev/) - set of date/time pickers for Material-UI. Also member of react-native-community and maintainer of [react-native-status-bar](https://github.com/react-native-community/react-native-statusbar). I love to make the software world better.
 
-In short: speaker, holistic open-source contributor. Typescript/Ocaml, react and react-native fan. Also volleyball player and [amateur photographer](https://www.instagram.com/dmtr._.kovalenkoo/) (ᵔᴥᵔ)
\ No newline at end of file
+In short: speaker, holistic open-source contributor. Typescript/Ocaml, react and react-native fan. Also volleyball player and [amateur photographer](https://www.instagram.com/dmtr._.kovalenkoo/) (ᵔᴥᵔ)
```
Comment (eps1lon) [inline content/about.md:5]: No idea why this appears as a change. Didn't touch that section.

Comment (dmtrKovalenko) [reply content/about.md:5]: Seems like a prettier :) No worries. Thank you a lot. I just didn't check it
